common/credCacheInternal_linux.go

```diff
@@ -105,7 +105,14 @@ func (c *CredCacheInternalIntegration) removeCachedTokenInternal() error {
 	return errors.New("Not implemented")
 }
 
+// saveTokenInternal persists an oauth token on disk.
```

Comment:
If not useful, why do we want to implement this function? Where is it used currently?

Reply:
So this is existing in our original workflow , right now its not used anywhere hence not implemented .

Reply:
changes for this module has been un-done.